focus-group.js

```diff
@@ -1,6 +1,12 @@
-export function menuKeyUX(options) {
+const Roles = {
```

Comment:
In my code style I use `ROLES`. Can you replace, please?

Reply:
Sure! My bad I missed it